src/helpers/utils.ts

```diff
@@ -208,3 +208,9 @@ export function captureError(e: any, context?: any, ignoredErrorCodes?: number[]
 export async function clearStampCache(type: string, id: string) {
   return fetch(`https://cdn.stamp.fyi/clear/${type}/${type === 'avatar' ? 'eth:' : ''}${id}`);
 }
+
+export function formatApp(app: string | undefined): string {
+  if (!app || app?.length > 128) return '';
```

Comment:
We should update on snapshot.js instead

https://github.com/snapshot-labs/snapshot.js/blob/master/src/schemas/proposal.json#L86